egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json

```diff
@@ -156,5 +156,110 @@
+    "chartName": "`DSS_HEALTH_OVERVIEW_POPULATION_COVERAGE_ACHIEVED`",
+    "queries": [
+      {
+        "module": "COMMON",
+        "dateRefField": "",
+        "requestQueryMap": "{\"province\": \"province\", \"district\": \"district\"}",
+        "indexName": "project-task-index-v1",
+        "aggrQuery": "{\n  \"aggs\": {\n    \"AGGR\": {\n      \"filters\": {\n        \"filters\": {\n          \"Household\": {\n            \"bool\": {\n              \"must\":[\n            {\n              \"term\": {\n                \"deliveredTo\": \"HOUSEHOLD\"\n              }\n            },\n            {\n              \"term\": {\n                \"productVariant\": \"LLIN\"\n              }\n            },\n            {\n              \"range\" : {\n                \"createdTime\" : {\n                  \"lte\": \"now\"\n                }\n              }\n            }\n            ]\n            }\n          }\n        }\n      }, \n      \"aggs\": {\n        \"SUM\": {\n          \"sum\": {\n            \"field\": \"quantity\"\n          }\n        },\n        \"Population Covered till Today\": {\n          \"bucket_script\": {\n            \"buckets_path\": {\n              \"sum\": \"SUM\"\n            },\n            \"script\": \"(params.sum) * 1.8\"\n          }\n        }\n      }\n    }\n  }\n}"
+      },
+      {
+        "module": "COMMON",
+        "dateRefField": "",
+        "requestQueryMap": "{\"province\": \"province\", \"district\": \"district\"}",
+        "indexName": "micro-plan-index-v1",
+        "aggrQuery": " {\n  \"aggs\": {\n    \"AGGR\": {\n      \"filters\": {\n        \"filters\": {\n          \"Campaign Filter\":{\n            \"bool\": {\n              \"must\" : [\n                {\n                  \"term\": {\n                    \"targetType\": \"INDIVIDUAL\"\n                  }\n                },\n                {\n                  \"term\": {\n                    \"productVariant\": \"LLIN\"\n                  }\n                }\n                ]\n            }\n          }\n        }\n        },\n      \"aggs\": {\n        \"Target per day\": {\n          \"sum\": {\n            \"field\": \"targetPerDay\"\n          }\n        },\n        \"Overall Target\": {\n          \"sum\": {\n            \"field\": \"overallTarget\"\n          }\n        },\n        \"Campaign Total Duration in days\": {\n          \"sum\": {\n            \"field\": \"campaignDurationInDays\"\n          }\n        },\n        \"Target Population per Day\": {\n          \"bucket_script\": {\n            \"buckets_path\": {\n              \"tpd\": \"Target per day\"\n            },\n            \"script\": \"(params.tpd) * 365\"\n          }\n        },\n        \"Overall Population Target\": {\n          \"bucket_script\": {\n            \"buckets_path\": {\n              \"opt\": \"Overall Target\"\n            },\n            \"script\": \"(params.opt) * 365\"\n          }\n        }\n      }\n      }\n    }\n  }"
```

Comment:
@bhanuprakash-egov As this is the overall target percentage, the day wise adjustment is not required for this query (i.e, target * 365 is not required)

Please refer the overall Household target query